addon/utils/cp/uses-transition.js

```diff
@@ -0,0 +1,7 @@
+import { computed } from '@ember/object';
+
+export default function usesTransition() {
+  return computed('fade', 'fastboot.isFastBoot', function() {
+    return !this.get('fastboot.isFastBoot') && this.get('fade');
+  })
```

Comment:
This is the only place where `fastboot` CP is consumed isn't it? Do we need to observe the `isFastBoot` property of `fastboot` service? I would expect that it can't change. Either the application runs in Fastboot or not. If that's the case we may could drop the `fastboot` cp and inline it's logic here. Would remove the dependencies between the CPs which could easily cause bugs.

Reply:
> This is the only place where fastboot CP is consumed isn't it? 

No, it's also used in `bs-modal`.

Reply:
I think an `isFastboot` utility function would serve both cases and reduce the magic:

```js
export default function(context) {
  let owner = getOwner(context);
  let fastbootService = owner.lookup('service:fastboot');
  return fastbootService ? fastbootService.get('isFastboot') : false;
}
```